### ISSUE TITLE
Display all applicants/members in a filterable table that accepts bulk operations like update/reject

### ISSUE THREAD

**Issue by ndewhurst** [orientier/ihc#10]

**Comment by ndewhurst** [comment]
Location filtering: each member will have a ZIP code - filter based on radius from ZIP code.

**Comment by ndewhurst** [comment]
Current "tiers" (based on total # of followers across channels):

- Beginner (<5k)
- Standard (5-15k)
- Premier (15-40k)
- Elite (>40k)

We should be able to automate tier assignment (i.e. initial assignment with possibility for admins to override).

**Comment by ndewhurst** [comment]
Note: the "badge added" checkbox indicates that an influencer has added a specific IHC "badge" image to her primary blog/webpage. We provide new members with an HTML snippet to add to their pages. In the future, we could automatically verify whether the badge is present.

**Comment by ndewhurst** [comment]
Note: the "badge added" checkbox indicates that an influencer has added a specific IHC "badge" image to her primary blog/webpage. We provide new members with an HTML snippet to add to their pages. In the future, we could automatically verify whether the badge is present.

**Comment by ndewhurst** [comment]
@meganshuffleton1 do we want to archive the applicants who we reject, or delete them from the system completely?

**Comment by mshuffleton** [comment]
@nathanaeldewhurst I would prefer to be able to archive them and keep their info on hand.

**Comment by ndewhurst** [comment]
@meganshuffleton1 ok! I think you let me know just in time, because I got the impression from @marissavalente3 in standup this morning that it was OK to delete them. At this point the functionality is pretty much built for either approach, so I'll finalize the archive option.

**Comment by mshuffleton** [comment]
Great! Apologies if there was a miscommunication. For reference, right now I vet applications and move girls into one of three tabs: onboarding, no, and not-right-now. To be honest I rarely go back and visit the no and non-right-now tabs, but it does give me peace of mind to have that info on hand in the case that we want to communicate with them or accept them in the future. So not a dealbreaker but prefer to be able to archive!

**Comment by ndewhurst** [comment]
@meganshuffleton1 ok. Would you like to have two different "non-approval" options available when reviewing applicants? E.g. "Reject" and "Reject for now (revisit in future)"...oslt? Or should we just have one "Reject" button and put all those rejected applicants in the same bucket/archive?

**Comment by mshuffleton** [comment]
@ndewhurst If that's easy for you to do, then yes that would be helpful! If it's much more complicated to do it that way, one reject button will suffice. Basically, these functions aren't imperative for me - so if there are some options on your end that just make more sense logistically, I trust your judgment.

**Comment by ndewhurst** [comment]
:thumbsup:

**Comment by ndewhurst** [comment]
@meganshuffleton1 @meganshuffleton1 I don't think we need to display email addresses in these tables, since most email communication will be automated (and admins can always click through to view an influencer/applicant's full profile and grab the email from there if needed).

**Comment by mshuffleton** [comment]
@nathanaeldewhurst Agreed if email comm will be automated through the platform! Otherwise I do sometimes grab multiple email addresses at once and send a bulk email. But if this will be possible through the platform, should be fine.
